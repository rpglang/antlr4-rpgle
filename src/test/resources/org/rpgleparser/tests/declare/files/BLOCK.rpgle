       dcl-f table1 DISK BLOCK(*YES);
       dcl-f table1 DISK BLOCK(*NO);       