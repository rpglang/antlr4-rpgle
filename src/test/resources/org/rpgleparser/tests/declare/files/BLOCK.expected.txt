/*===TOKENS===*/
FS_FreeFile              <dcl-f>
ID                       <table1>
KEYWORD_DISK             <DISK>
KEYWORD_BLOCK            <BLOCK>
'('                      <(>
SPLAT_YES                <*YES>
')'                      <)>
FREE_SEMI                <;>
FS_FreeFile              <dcl-f>
ID                       <table1>
KEYWORD_DISK             <DISK>
KEYWORD_BLOCK            <BLOCK>
'('                      <(>
SPLAT_NO                 <*NO>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (fspec
      dcl-f
      (filename table1)
      (fs_keyword (keyword_disk DISK))
      (fs_keyword (keyword_block BLOCK ( (symbolicConstants *YES) )))
      ;
    )
  )
  (statement
    (fspec
      dcl-f
      (filename table1)
      (fs_keyword (keyword_disk DISK))
      (fs_keyword (keyword_block BLOCK ( (symbolicConstants *NO) )))
      ;
    )
  )
)
/*======*/